rustls/src/crypto/ring/ticketer.rs

```diff
@@ -69,6 +71,10 @@ struct AeadTicketer {
     alg: &'static aead::Algorithm,
     key: aead::LessSafeKey,
     lifetime: u32,
+
+    /// Tracks the largest ciphertext produced by `encrypt`, and
+    /// uses it to early-reject `decrypt` queries that are too long.
+    maximum_ciphertext_len: AtomicUsize,
```

Comment:
Can we fold in a short explanation about why a peer might be sending too-long-ciphertexts? E.g. something about avoiding allowing a partitioning oracle to be constructed? I think it could be helpful to motivate the control being added.

Reply:
New text is exactly what I was looking for :+1:  Thanks!